### PR TITLE
GB28181: fixed compile error when --log-info=on

### PR DESCRIPTION
**Pull request by osenberg-x**
fixed compile error when --log-info=on

### CODE REVIEW

**Comment by winlinvip** [comment]
https://github.com/ossrs/srs/pull/2506/commits/3c2e35f8b91246a0a2d71e3ed0bbfbfb389bfadb
Just take a look at this commit.

`TRANS_BY_GPT3`

**Review by xialixin — APPROVED** [review]

File: trunk/src/app/srs_app_gb28181.cpp [hunk 1]
```diff
@@ -782,8 +782,10 @@ srs_error_t SrsPsStreamDemixer::on_ps_stream(char* ps_data, int ps_size, uint32_
 
                 srs_error_t err2 = srs_success;
                 if ((err2 = aac->adts_demux(&avs, &frame, &frame_size, codec)) != srs_success) {
+                    // srs_info("gb28181: client_id %s, audio data not aac adts (%#x/%u) %02x %02x %02x %02x\n", 
+                    //          channel_id.c_str(), ssrc, timestamp, p1, p2, p3, p4);  
                     srs_info("gb28181: client_id %s, audio data not aac adts (%#x/%u) %02x %02x %02x %02x\n",
```
Comment (duiniuluantanqin) [inline trunk/src/app/srs_app_gb28181.cpp:787]: `%02x %02x %02x %02x`，这里有4个参数，如果要删掉p3,p4,应该减少2个参数吧。

File: trunk/src/app/srs_app_gb28181.cpp [hunk 2]
```diff
@@ -782,8 +782,10 @@ srs_error_t SrsPsStreamDemixer::on_ps_stream(char* ps_data, int ps_size, uint32_
 
                 srs_error_t err2 = srs_success;
                 if ((err2 = aac->adts_demux(&avs, &frame, &frame_size, codec)) != srs_success) {
+                    // srs_info("gb28181: client_id %s, audio data not aac adts (%#x/%u) %02x %02x %02x %02x\n", 
+                    //          channel_id.c_str(), ssrc, timestamp, p1, p2, p3, p4);  
                     srs_info("gb28181: client_id %s, audio data not aac adts (%#x/%u) %02x %02x %02x %02x\n", 
-                             channel_id.c_str(), ssrc, timestamp, p1, p2, p3, p4);  
+                             channel_id.c_str(), ssrc, timestamp, p1, p2);
```
Comment (duiniuluantanqin) [inline trunk/src/app/srs_app_gb28181.cpp:786]: %02x %02x %02x %02x, there are 4 parameters here, if you want to delete p3, p4, you should reduce 2 parameters, right?

`TRANS_BY_GPT3`

**Review by duiniuluantanqin — APPROVED** [review]

File: trunk/src/app/srs_app_gb28181.cpp (hunk 2 above)
Comment (duiniuluantanqin) [reply trunk/src/app/srs_app_gb28181.cpp:786]: I made modifications based on this.

`TRANS_BY_GPT3`

**Review by xialixin — APPROVED** [review]

**Comment by duiniuluantanqin** [comment]
@osenberg-x Sorry, could you please delete my commit or submit a new pull request? Otherwise, it will appear as if I made the submission.

`TRANS_BY_GPT3`

**Review by duiniuluantanqin — APPROVED** [review]

**Comment by winlinvip** [comment]
Merged as ceb13b58d

**Pull request closed without merging**